bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF

```diff
@@ -15,4 +15,6 @@ Import-Package: groovy.lang,
  org.eclipse.smarthome.core.library.types,
  org.eclipse.smarthome.test,
  org.hamcrest;core=split,
- org.junit;version="4.0.0"
+ org.junit;version="4.0.0",
+ org.junit.runner;version="4.11.0",
+ org.junit.runners;version="4.11.0"
```

Comment:
Is there any reason you need at least 4.11?
IMHO we should use 4.0.0 only.
We already removed the usage of 4.5.0, 4.8.0 etc.
e.g.
https://github.com/eclipse/smarthome/pull/439/files#diff-598bdbf2a5128a7949ce6ed2c21bb436L19